src/definitions/elements/step.less

```diff
@@ -153,9 +153,6 @@
   flex: 0 1 auto;
   align-self: @iconAlign;
 }
-.ui.steps .step > .icon ~ .content {
```

Comment:
Why was this removed?

Reply:
The rule above was a duplicate as `flex-grow` is a part of the `flex` property.

Reply:
Ok cool